### PR TITLE
Mention disabling remote cache for Groovy build script compilation in release notes

### PR DESCRIPTION
**Pull request by asodja**

### CODE REVIEW

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -447,8 +450,12 @@ plugins {
 }
 ```
 
-The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
-If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
+#### Groovy build script compilation build cache support is disabled
+In Gradle 8.7 we announced that [Groovy build script](userguide/groovy_build_script_primer.html) compilation benefits from the remote build cache.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:454]:
```suggestion
In Gradle 8.7, we introduced support for using the remote build cache with [Groovy build script](userguide/groovy_build_script_primer.html) compilation.
```

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -96,6 +96,9 @@ beforeSettings {
 }
 ```
 
+The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:99]: @asodja, is this related?

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 3]
```diff
@@ -447,8 +450,12 @@ plugins {
 }
 ```
 
-The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
-If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
+#### Groovy build script compilation build cache support is disabled
+In Gradle 8.7 we announced that [Groovy build script](userguide/groovy_build_script_primer.html) compilation benefits from the remote build cache.
+Due to reports from users about slower compile times with a remote cache, we did further investigation and found that the cache was not providing the expected performance improvements.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:455]:
```suggestion
However, after receiving reports of slower compile times with the remote cache, we conducted further investigation. 
Our findings showed that the cache was not delivering the expected performance improvements.
```

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 4]
```diff
@@ -447,8 +450,12 @@ plugins {
...
+
+We disabled the remote build cache for Groovy build script compilation in this release.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:457]:
```suggestion
As a result, we have disabled the remote build cache for Groovy build script compilation in this release.
```

**Review by lkasso — CHANGES_REQUESTED** [review]

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 5]
```diff
@@ -96,6 +96,9 @@ beforeSettings {
 }
 ```
 
+The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
+If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:100]:
```suggestion
If this setting is configured with a non-default value, it will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
```

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 6]
```diff
@@ -447,8 +450,12 @@ plugins {
 }
 ```
 
-The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
-If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
+#### Groovy build script compilation build cache support is disabled
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:453]: @ljacomet should this be in "Other improvements" or "Build Cache improvements"?
Should these changes be made to the release branch or in the Google doc?

Comment (asodja) [reply platforms/documentation/docs/src/docs/release/notes.md:453]: This deprecations were before under `#### Allow version catalog plugin aliases without a version`, but they actually relate to change described under `#### Improved control of local build cache cleanup`. 

So I moved it to that section.

**Comment by ljacomet** [comment]
Moving the proposed changes to our working document for release notes.

Closing this PR.
